.circleci/config.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+jobs:
+  build:
+      
+      docker:
+        - image: circleci/openjdk:8-jdk-browsers
+        
+      steps:   
+        - checkout
```

Comment:
Z tego co widzę, to na circle ci ten kod został pobrany, ale testy nie były uruchomione?

Reply:
Tak, zgadza się. Niestety nie mogłem sobie poradzić na circle ci z surfire plugin. Projekt zbudował się na circle ci, ale bez testów.